run/logging-manual/main.go

```diff
@@ -78,6 +78,12 @@ func (e Entry) String() string {
 // [END run_manual_logging_object]
 // [START run_manual_logging]
 
+func init() {
+	// Disable log timestamps, these are added automatically.
+	// Prefix text leads to malformed JSON.
```

Comment:
Sorry I missed this earlier -- the comment isn't clear to me. What adds timestamps automatically? What JSON?

Reply:
The structured logging the sample produces is a JSON object. If the timestamp prefix is added, we end up with something like this:

```
2020/01/05 15:41:22 {"message":"Storing link in /dev/null"}
```

This is read as plain text, not JSON, and as a result the structure is not processed for Stackdriver.

Reply:
Will you update the comment? Not sure the best way to phrase it. "These" is ambiguous in the first sentence. And maybe expand the second sentence?

Reply:
What about this?

```
// Disable log prefixes such as the default timestamp.
// Prefix text prevents the message from being parsed as JSON.
// A timestamp is added by the process of shipping logs to Stackdriver.
```

Reply:
"... in the process ..."?

SGTM.

Reply:
Going to change to this: 

// Disable log prefixes such as the default timestamp.
// Prefix text prevents the message from being parsed as JSON.
// A timestamp is added when shipping logs to Stackdriver.